config.py

```diff
@@ -36,7 +36,7 @@ class Config(object):
     JSON_SECRET_KEYS = os.getenv('JSON_SECRET_KEYS')
 
     REDIS_HOST = os.getenv('REDIS_HOST', 'localhost')
-    REDIS_PORT = os.getenv('REDIS_PORT', 6379)
```

Comment:
Changing this back to 6379 should make it work in travis. Looks like you've changed it to make the test pass locally so changing it back should work